private-templates-service/server/dist/app.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
I guess you should ignore dist folder(or all generated files)

Reply:
Just added dist/ to the .gitignore folder.

Reply:
Good calling adding those to gitignore. If they were already commited into the repo you will have to `git rm` them to manually delete them; since gitignore only applies to new files

Reply:
I just removed all files under dist/.